packages/react/src/patterns/sub-patterns/ContentItem/ContentItem.js

```diff
@@ -40,16 +42,28 @@ const ContentItem = ({ copy, heading, image }) => (
         data-autoid={`${stablePrefix}--content-item__copy`}
         className={`${prefix}--content-item__copy`}
         dangerouslySetInnerHTML={{
-          __html: markdownToHtml(copy),
+          __html: markdownToHtml(copy, { bold: false }),
         }}></div>
     )}
+    {cta && <CTA style="text" type={cta.type} copy={cta.copy} />}
```

Comment:
I wonder if it makes sense to add this logic to the CTA component. Thoughts?

Reply:
hmm, might be overdoing it, should just see `cta` as its own component that can be used wherever. If we added a null check on the root level, would we do that for every component we make?

Reply:
I would think we should. I guess it depends on the experience we want to build for our adopters. Do we want to throw a nasty error if the data is missing and incorrect, or do we just not render a component if the data is malformed. I've generally leaned towards the later throughout most of my development practice because then the application is at least still usable, but I can see a case for throwing errors too.

Reply:
Not throwing errors in this case would make it tough for folks to debug I would think. It's also a new practice we haven't applied to our other components. Not to mention, it helps our own development when we're building out decorator patterns that make use of these sub-patterns and components, and making sure that we are utilizing them correctly. 

I think throwing errors is something we should keep in place.